web/src/components/forms/components/pep-search-dropdown.tsx

```diff
@@ -65,6 +65,13 @@ export const PepSearchDropdown = (props: Props) => {
           value: `${n.namespace}/${n.name}:${n.tag}`,
         })) || []
       }
+      styles={{
```

Comment:
Just curious if this is a common thing we add to dropdowns. If so, could add to a consts file and then import everywhere we use it